### ISSUE TITLE
Why is it getting slower when  using nn.graph

### ISSUE THREAD

**Issue by f291400** [Oneflow-Inc/oneflow#10259]
## NetWork Defintion
![image](https://github.com/Oneflow-Inc/oneflow/assets/13358476/0dc886a6-775a-4ab0-a3a0-649fed00b4b5)

def build(self, dic, seq_len=2048, record=0, save=False):
       answer, history = self.model.stream_chat(self.tokenizer, input_dict, history=[], max_new_tokens=seq_len, do_sample=False)

## Ques
when setting config.enable_amp(True), self.config.allow_fuse_cast_scale(True),self.config.enable_straighten_algorithm('SpeedFirst')..., time cost increase from 0.4s to 1.2s， it runs slower.

**Comment by strint** [comment]
The first call of nn.Graph will do graph compilation which tasks time. Did you ignore the time of the first call?

**Comment by f291400** [comment]
Yes, I  compare the interface cost time after ten times of warm-up running.

![image](https://github.com/Oneflow-Inc/oneflow/assets/13358476/9e585bb6-cc43-46c2-baed-2974ac19dc26)

when I set debug(0), there is no special log printed on the screen
![image](https://github.com/Oneflow-Inc/oneflow/assets/13358476/18ce6fd7-c89e-4c8b-b0bf-dd5cdeea4849)

**Comment by strint** [comment]
> when I set debug(0), there is no special log printed on the screen

Try to set debug(2)，logs are expected to show at the first call of nn.Graph.


> when setting config.enable_amp(True), self.config.allow_fuse_cast_scale(True),self.config.enable_straighten_algorithm('SpeedFirst')..., time cost increase from 0.4s to 1.2s， it runs slower.

On the speed issue,  did it get slow because you enable these configs? 

You can try to open/close one of these config to see which one affect the speed.

**Comment by f291400** [comment]
> On the speed issue, did it get slow because you enable these configs?
I test again.  I found  it runs slower, because using 'torch.nn.Graph'
![image](https://github.com/Oneflow-Inc/oneflow/assets/13358476/022eeccb-b900-4834-b999-5534f16a391c)
Not use torch.nn.Graph  cost 0.7s
Using torch.nn.Graph without 'self.config' cost 1.2s

**Comment by strint** [comment]
Is your code on github？We need to see the code to figure out what is happening.

**Comment by f291400** [comment]
Our model code is forked from chatglm-6b.https://huggingface.co/THUDM/chatglm-6b
